src/components/measurements/index.tsx

```diff
@@ -25,24 +25,40 @@ import {
   layout,
   jitterRawMeansByColorBy
 } from "./measurementsD3";
+import { RootState } from "../../store";
+import { MeasurementFilters } from "../../reducers/controls";
+import { Visibility } from "../../reducers/tree/types";
+import { Measurement } from "../../reducers/measurements/types";
+
+interface TreeStrainColors {
+  [strain: string]: {
+    attribute: string
+    color: string
+  }
+}
+interface TreeStrainVisibility {
+  [strain: string]: Visibility
+}
+interface TreeStrainProperties {
+  treeStrainVisibility: TreeStrainVisibility
+  treeStrainColors: TreeStrainColors
+}
```

Comment:
(nit)

If `TreeStrainColors` is not used elsewhere, the intermediate alias can be removed:


```suggestion
interface TreeStrainProperties {
  treeStrainVisibility: TreeStrainVisibility
  treeStrainColors: {
    [strain: string]: {
      attribute: string
      color: string
    }
  }
}
```

Reply:
Fixed up in https://github.com/nextstrain/auspice/pull/1910/commits/f5b3f2fb1d744abeade9383ed683219ef98b9cec